### PR TITLE
Add user @flying-sheep, update recipe to 1.4

### PR DESCRIPTION
**Pull request by flying-sheep**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)

### CODE REVIEW

**Comment by flying-sheep** [comment]
@conda-forge/core please merge, @raivivek hasn’t updated this in a while

**Comment by ocefpaf** [comment]
@flying-sheep if you send a PR fixing the recipe and adding yourself, it will make it easier for the core members to merge it without waiting for the maintainers.

**Comment by flying-sheep** [comment]
I just did that

**Comment by ocefpaf** [comment]
> I just did that. This is the PR where I add myself.

I'm in automate mode here reading tons of notifications, sorry!

**Comment by ocefpaf** [comment]
Actually, it only adds you. Can you update it to 1.3, that is what I meant. Otherwise this is the same as the not one.

> https://github.com/Techtonique/ahead_python/blob/v0.10.0/setup.py

Sure, but we have 1-week wait time for the maintainer to answer. IF you update the recipe and add yourself, we can speed up the process and merge it right away.

**Comment by flying-sheep** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {% set name = "legacy-api-wrap" %}
-{% set version = "1.2" %}
+{% set version = "1.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml:9]: Sadly, PyPI is not consistent with the `-` and `_`.

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/legacy_api_wrap-{{ version }}.tar.gz
```

Comment (flying-sheep) [reply recipe/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name | replace("-", "_") }}-{{ version }}.tar.gz
```

**Comment by flying-sheep** [comment]
Hi @ocefpaf, why did merging this in not publish a new version?

**Comment by ocefpaf** [comment]
It did work but Anaconda's CDN seems to be very slow lately. I'm getting tons of:

Screenshot from 2024-03-27 18-15-57


However, after refreshing a few times I did see version 1.4 there.

**Comment by flying-sheep** [comment]
OK, thank you!

**Pull request merged**